syntax = "proto3";

package loadbalancer;
import "google/protobuf/empty.proto";

service LoadBalancerService {
    rpc HandleRequest (Request) returns (Response) {}
}

message Request {
    string message = 1;
}

message Response {
    string message = 1;
    string server_id = 2;
}
